internal/mode/static/state/graph/policies.go

```diff
@@ -179,8 +182,10 @@ func processPolicies(
 					continue
 				}
 			case hrGroupKind, grpcGroupKind:
-				if _, exists := routes[routeKeyForKind(ref.Kind, refNsName)]; !exists {
+				if route, exists := routes[routeKeyForKind(ref.Kind, refNsName)]; !exists {
 					continue
+				} else {
```

Comment:
you don't need this else statement

Reply:
I do because the `route` variable is defined in the context of the if block. Otherwise, I would need to define it outside the if block.